src/app/actions/knowledge.ts

```diff
@@ -0,0 +1,12 @@
+import { Action } from '@ngrx/store';
+import { type } from '../utils';
+export const ActionTypes = {
+  CHANGE: type('[Knowledge] Change')
```

Comment:
Make it --> ```CHANGE: '[Knowledge] Change'```.
No need to define type here if you are defining type in the below implementation class. Do check its successful working though.

Reply:
It is working but removing that is decreasing coverage. Typescript is strongly typed, I think this is the reason coverage is decreasing. This approach is followed throughout the project, so we can leave it like that.

Reply:
It is working but removing that is decreasing coverage. Typescript is strongly typed, I think this is the reason coverage is decreasing. This approach is followed throughout the project, so we can leave it like that.